jobsdb/jobsdb.go

```diff
@@ -3557,6 +3557,9 @@ func (jd *HandleT) backupUploadWithExponentialBackoff(ctx context.Context, file
 	if err != nil {
 		return filemanager.UploadOutput{}, err
 	}
+
+	defer fileUploader.Dispose()
```

Comment:
```suggestion
```
`jobsdb.HandleT` is supposed to have a `jobsFileUploader` field for caching a single instance of a `filemanager.FileManager`. Disposing a cached fileUploader every time this method gets called is not desirable.

Btw, after looking at `getFileUploader()` method, I believe that it needs some changes... e.g.
```go
// getFileUploader get a file uploader
func (jd *HandleT) getFileUploader() (filemanager.FileManager, error) {
	var err error
	if jd.jobsFileUploader == nil {
		jd.jobsFileUploader, err = filemanager.DefaultFileManagerFactory.New(&filemanager.SettingsT{
			Provider: config.GetEnv("JOBS_BACKUP_STORAGE_PROVIDER", "S3"),
			Config:   filemanager.GetProviderConfigFromEnv(),
		})
	}
	return jd.jobsFileUploader, err
}
```

Reply:
Good catch. Fixed.